src/App.js

```diff
@@ -230,11 +229,8 @@ export default function App(props) {
   }, [urlParam]);
 
   function resetLogMessages() {
-    consoleMessages = [];
-    errorString = '';
-    warningString = '';
-    errorCount = 0;
-    warningCount = 0;
+    infoMessages = [defaultInfoMessage];
+    problemMessages = [defaultProblemMessage];
```

Comment:
Believe `problemCount` needs to be reset here.

Reply:
Addressed in `4388dc0`.